### ISSUE TITLE
Update SampleActivity to use a single persistent Service connection

### ISSUE THREAD

**Issue by wdziemia** [NYTimes/Register#48]

**Comment by wdziemia** [comment]
This is to fix a new service connection for every interaction w/ Register.

**Comment by candidcold** [comment]
I don't think this is applicable anymore since the update to Play Billing, correct me if I'm wrong @wdziemia @goldstei 

We may be able to close this
